src/routes/resources/+page.svelte

```diff
@@ -271,10 +285,22 @@
   </form>
 </Collapsible>
 
+<ol
+  class="grid xl:grid-cols-3 md:grid-cols-2 sm:grid-cols-1 gap-x-4 gap-y-4 mt-3"
+>
+  {#each displayedResources.slice(0, displayedResourceLimit) as resource}
+    <li><ListItem {resource} /></li>
+  {:else}
+    <li>No resources here!</li>
+  {/each}
+</ol>
+<div class="italic text-center m-4">Those are all the resources!</div>
+
 <button
-  class="w-10 h-10 inline-flex items-center justify-center rounded-full bg-green-500 text-white cursor-pointer fixed ease-in-out bottom-10 right-10 z-50"
+  class="w-10 h-10 inline-flex items-center justify-center rounded-full bg-green-700 dark:bg-green-900/75 text-white cursor-pointer fixed transition-opacity bottom-10 right-10 z-50 outline-offset-2 {showButton
+    ? 'opacity-100'
+    : 'opacity-0'}"
   on:click={scrollToTop}
-  class:hidden={!showButton}
```

Comment:
What was the motivation for this specific refactor? It still works, so no need to revert, just curious